atlast_sc/derived_groups.py

```diff
@@ -25,15 +25,17 @@ def __init__(self):
 
         T_atm_table = np.genfromtxt(AtmosphereParams._T_ATM_PATH)
         tau_atm_table = np.genfromtxt(AtmosphereParams._TAU_ATM_PATH)
-        # TODO: interp2d is deprecated:
-        #   see https://docs.scipy.org/doc/scipy/reference/generated
-        #       /scipy.interpolate.interp2d.html
-        self._interp_T_atm = interp2d(T_atm_table[:, 0],
-                                      AtmosphereParams._WEATHER,
-                                      T_atm_table[:, 1:].T)
-        self._interp_tau_atm = interp2d(tau_atm_table[:, 0],
-                                        AtmosphereParams._WEATHER,
-                                        tau_atm_table[:, 1:].T)
+
```

Comment:
Could we add a comment here to explain this?
```suggestion
        # the temperature values obtained by interpolating over the ATM tables are rescaled by the opacity at zenith to obtain T_atm (see the discussion around Eq. 7-9 in the ALMA Memo 602 (https://library.nrao.edu/public/memos/alma/main/memo602.pdf))
```